mlir/lib/Dialect/Rock/IR/RockDialect.cpp

```diff
@@ -655,10 +667,10 @@ GemmSize Conv2DBwdDataOp::getGemmSize() {
   int64_t yTilda = strideH / gcdStrideDilationH;
```

Comment:
Have not turned some of these variables into arrays and thus have no loops for the dimensions.

Reply:
These _should_ generalize out nicely? But that's a future PR, I think

Reply:
And, right, this is a different file and we're probably looking at a struct that could be unified away